app/backend/src/tests/test_fixtures.py

```diff
@@ -406,3 +431,8 @@ def fast_verify(hashed: bytes, password: bytes) -> bool:
     with patch("couchers.crypto.nacl.pwhash.verify", fast_verify):
         with patch("couchers.crypto.nacl.pwhash.str", fast_hash):
             yield
+
+
+def lists_equal(list1, list2):
+    """Compare equality of (hashable) elements in two lists, by computing size of symmetric set difference"""
+    return len(set(list1) ^ set(list2)) == 0
```

Comment:
Remove function and inline into callers instead - I've commented on some examples.